js/modules/Forms/RendererController.js

```diff
@@ -66,7 +66,7 @@ export class GlpiFormRendererController
      */
     constructor(target, form_id) {
         // Target must be a valid form
-        this.#target = target;
+        this.#target = document.querySelector(target);
```

Comment:
Change to a real HTMLFormElement instead of a raw selector for convenience when the code doesn't need jquery.